framework-plugins/lisk-framework-http-api-plugin/src/controllers/transactions.ts

```diff
@@ -109,24 +109,23 @@ export const postTransaction = (channel: BaseChannel, codec: PluginCodec) => asy
 		return;
 	}
 
-	const encodedTransaction = codec.encodeTransaction(req.body as TransactionInput);
-
-	const result = await channel.invoke<{
-		transactionId?: string;
-		message?: string;
-		errors?: Error[] | Error;
-	}>('app:postTransaction', {
-		transaction: encodedTransaction,
-	});
+	try {
+		const encodedTransaction = codec.encodeTransaction(req.body as TransactionInput);
 
-	if (result.errors) {
-		const processingErrors = (result.errors as Error[]).map(e => ({
-			message: e.message,
-		}));
+		const result = await channel.invoke<{
+			transactionId?: string;
+			message?: string;
+			errors?: Error[] | Error;
+		}>('app:postTransaction', {
+			transaction: encodedTransaction,
+		});
 
-		res.status(409).json({ errors: processingErrors });
-		return;
+		res.status(200).json({ data: result, meta: {} });
+	} catch (err) {
+		res.status(409).json({
+			errors: [
+				{ message: (err as Error).message.replace('Lisk validator found 1 error[s]:\n', '') },
```

Comment:
I think we don't need to add extra logic to filter the error message here.
it can be `[ err ] ` IMO. If the output is wrong, we should fix in the framework side

Reply:
To fix in framework side we need to write a new handler there I think

Reply:
As discussed the lisk-validator handling will be checked as a separate issue.